app/src/main/java/timur/gilfanov/messenger/ui/screen/user/LanguageUiState.kt

```diff
@@ -11,6 +12,6 @@ import kotlinx.collections.immutable.ImmutableList
  * @property selectedLanguage The currently selected language
  */
 data class LanguageUiState(
```

Comment:
Must add @Immutable or Compose may treat LanguageUiState as unstable, causing unnecessary recompositions every time the parent recomposes:
 1. UiLanguage is a sealed interface - Compose compiler doesn't automatically recognize it as stable
 2. ImmutableList from kotlinx-collections-immutable is NOT automatically recognized as stable by the Compose compiler

 Alternative: Configure stability globally in your Compose compiler options via a stability configuration file:

  // stability_config.conf
  kotlinx.collections.immutable.*

  This tells Compose that all kotlinx immutable collections are stable, reducing the need for manual annotations.

```suggestion
@Immutable
data class LanguageUiState(
```